dev/docker/plugins/bedrock_plugin.hardlink.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/python3
```

Comment:
Can we simply symlink to the actual python script?

Reply:
It didn't seem to work in the docker image with a relative softlink.

Reply:
Removed the hardlinks. Just get devs to symlink manually for dev.